src/client.ts

```diff
@@ -73,9 +74,18 @@ export default class Client {
 
   private async negotiate(description: RTCSessionDescriptionInit) {
     try {
-      await this.pc.setRemoteDescription(description); // SRD rolls back as needed
-      await this.pc.setLocalDescription();
-      this.signal.answer(this.pc.localDescription!);
+      if (description.type === "offer" && (this.makingOffer || this.pc.signalingState !== "stable")) {
```

Comment:
Firs time I see a linter complaining about triple equals, it should be the standar to avoid weird js bugs